packages/@lwc/style-compiler/src/__tests__/index.spec.ts

```diff
@@ -60,26 +48,15 @@ function testFixture(fixtureDir: string) {
             const { code } = transform(actualSource, actualPath, config);
             result = code;
         } catch (err) {
-            if (expectedSource) {
-                throw err;
-            } else {
-                error = normalizeError(err);
-            }
+            error = normalizeError(err);
         }
 
-        if (expectedSource) {
-            expect(result).toBe(expectedSource);
-
+        if (error) {
+            expect(error).toMatchSnapshot();
+        } else {
+            expect(result).toMatchSnapshot();
```

Comment:
I considered checking if `result` was falsy when `error` is truthy and vice-versa, but based on the `try/catch` logic this seems unnecessary.